app/models/statistic.rb

```diff
@@ -0,0 +1,4 @@
+class Statistic < ApplicationRecord
```

Comment:
I suggest calling this `CourseStatistics` or `CourseStats`, since it's specifically designed to hold stats for a Course.

Reply:
Also, it should be in `app/models/course_data/`.